{
  "uid" : "3a882b1f9f34fa4f",
  "name" : "TC 10.04.05 - Verify the text in 2 columns block - Men's Deals",
  "fullName" : "tests.test_sale.TestPromoBlocks#test_tc_10_04_05",
  "historyId" : "0c929b279299e32eaa8a93ebe20ddfe0",
  "time" : {
    "start" : 1729997928458,
    "stop" : 1729997928458,
    "duration" : 0
  },
  "description" : "Check title-text and content-text in 2 columns block - Men's Deals",
  "descriptionHtml" : "<p>Check title-text and content-text in 2 columns block - Men's Deals</p>\n",
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.782\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x5600a46410aa <unknown>\n#1 0x5600a41581a0 <unknown>\n#2 0x5600a4140840 <unknown>\n#3 0x5600a414052d <unknown>\n#4 0x5600a413e5bc <unknown>\n#5 0x5600a413ed6f <unknown>\n#6 0x5600a414e1d7 <unknown>\n#7 0x5600a41641e4 <unknown>\n#8 0x5600a416951b <unknown>\n#9 0x5600a413f47e <unknown>\n#10 0x5600a4163fee <unknown>\n#11 0x5600a41e7acb <unknown>\n#12 0x5600a41c8b23 <unknown>\n#13 0x5600a4197990 <unknown>\n#14 0x5600a419896e <unknown>\n#15 0x5600a460d16b <unknown>\n#16 0x5600a4610f68 <unknown>\n#17 0x5600a45fa64c <unknown>\n#18 0x5600a4611ae7 <unknown>\n#19 0x5600a45df4af <unknown>\n#20 0x5600a462f4f8 <unknown>\n#21 0x5600a462f6c0 <unknown>\n#22 0x5600a463ff26 <unknown>\n#23 0x7f42fdc94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"654c65a8599441c25dc02aa8948f6b63\")>\n\n    @pytest.fixture(scope=\"function\")\n    def sale_page(driver):\n        \"\"\"This function navigate to Sale page and open it for testing\"\"\"\n        sale_page = SalePage(driver, SALE_PAGE_URL)\n>       sale_page.open()\n\ntests/test_sale.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/base_page.py:19: in open\n    self.driver.get(self.url)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fc6edf03910>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"timeout\",\"message\":\"timeout: Timed out receiving message from renderer: 2...\\\\n#21 0x5600a462f6c0 \\\\u003Cunknown>\\\\n#22 0x5600a463ff26 \\\\u003Cunknown>\\\\n#23 0x7f42fdc94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.782\nE         (Session info: chrome=130.0.6723.58)\nE       Stacktrace:\nE       #0 0x5600a46410aa <unknown>\nE       #1 0x5600a41581a0 <unknown>\nE       #2 0x5600a4140840 <unknown>\nE       #3 0x5600a414052d <unknown>\nE       #4 0x5600a413e5bc <unknown>\nE       #5 0x5600a413ed6f <unknown>\nE       #6 0x5600a414e1d7 <unknown>\nE       #7 0x5600a41641e4 <unknown>\nE       #8 0x5600a416951b <unknown>\nE       #9 0x5600a413f47e <unknown>\nE       #10 0x5600a4163fee <unknown>\nE       #11 0x5600a41e7acb <unknown>\nE       #12 0x5600a41c8b23 <unknown>\nE       #13 0x5600a4197990 <unknown>\nE       #14 0x5600a419896e <unknown>\nE       #15 0x5600a460d16b <unknown>\nE       #16 0x5600a4610f68 <unknown>\nE       #17 0x5600a45fa64c <unknown>\nE       #18 0x5600a4611ae7 <unknown>\nE       #19 0x5600a45df4af <unknown>\nE       #20 0x5600a462f4f8 <unknown>\nE       #21 0x5600a462f6c0 <unknown>\nE       #22 0x5600a463ff26 <unknown>\nE       #23 0x7f42fdc94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "sale_page",
    "time" : {
      "start" : 1729997929724,
      "stop" : 1729998229742,
      "duration" : 300018
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.782\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x5600a46410aa <unknown>\n#1 0x5600a41581a0 <unknown>\n#2 0x5600a4140840 <unknown>\n#3 0x5600a414052d <unknown>\n#4 0x5600a413e5bc <unknown>\n#5 0x5600a413ed6f <unknown>\n#6 0x5600a414e1d7 <unknown>\n#7 0x5600a41641e4 <unknown>\n#8 0x5600a416951b <unknown>\n#9 0x5600a413f47e <unknown>\n#10 0x5600a4163fee <unknown>\n#11 0x5600a41e7acb <unknown>\n#12 0x5600a41c8b23 <unknown>\n#13 0x5600a4197990 <unknown>\n#14 0x5600a419896e <unknown>\n#15 0x5600a460d16b <unknown>\n#16 0x5600a4610f68 <unknown>\n#17 0x5600a45fa64c <unknown>\n#18 0x5600a4611ae7 <unknown>\n#19 0x5600a45df4af <unknown>\n#20 0x5600a462f4f8 <unknown>\n#21 0x5600a462f6c0 <unknown>\n#22 0x5600a463ff26 <unknown>\n#23 0x7f42fdc94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/tests/test_sale.py\", line 17, in sale_page\n    sale_page.open()\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 19, in open\n    self.driver.get(self.url)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open a browser",
      "time" : {
        "start" : 1729997929724,
        "stop" : 1729998229742,
        "duration" : 300018
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.782\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x5600a46410aa <unknown>\n#1 0x5600a41581a0 <unknown>\n#2 0x5600a4140840 <unknown>\n#3 0x5600a414052d <unknown>\n#4 0x5600a413e5bc <unknown>\n#5 0x5600a413ed6f <unknown>\n#6 0x5600a414e1d7 <unknown>\n#7 0x5600a41641e4 <unknown>\n#8 0x5600a416951b <unknown>\n#9 0x5600a413f47e <unknown>\n#10 0x5600a4163fee <unknown>\n#11 0x5600a41e7acb <unknown>\n#12 0x5600a41c8b23 <unknown>\n#13 0x5600a4197990 <unknown>\n#14 0x5600a419896e <unknown>\n#15 0x5600a460d16b <unknown>\n#16 0x5600a4610f68 <unknown>\n#17 0x5600a45fa64c <unknown>\n#18 0x5600a4611ae7 <unknown>\n#19 0x5600a45df4af <unknown>\n#20 0x5600a462f4f8 <unknown>\n#21 0x5600a462f6c0 <unknown>\n#22 0x5600a463ff26 <unknown>\n#23 0x7f42fdc94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 19, in open\n    self.driver.get(self.url)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_result_folder",
    "time" : {
      "start" : 1729993092064,
      "stop" : 1729993092065,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1729993092000,
      "stop" : 1729993092064,
      "duration" : 64
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1729997928459,
      "stop" : 1729997929724,
      "duration" : 1265
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "clear_allure_result_folder::0",
    "time" : {
      "start" : 1729999808072,
      "stop" : 1729999808072,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "driver::0",
    "time" : {
      "start" : 1729998229767,
      "stop" : 1729998229873,
      "duration" : 106
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Sale Page"
  }, {
    "name" : "feature",
    "value" : "Testing Promo Blocks, Main Block (Women's Deals) and 2 columns block - Men's Bargain, Gear Steals"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_sale"
  }, {
    "name" : "subSuite",
    "value" : "TestSalePage"
  }, {
    "name" : "host",
    "value" : "fv-az841-46"
  }, {
    "name" : "thread",
    "value" : "2812-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_sale"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "3a882b1f9f34fa4f.json",
  "parameterValues" : [ ]
}